frontend/src/util/worksheet_utils.js

```diff
@@ -66,27 +66,42 @@ export function renderDate(epochSeconds) {
     return dt.toDateString() + ' ' + padInt(hour, 2) + ':' + padInt(min, 2) + ':' + padInt(sec, 2);
 }
 
-export function renderSize(size, includeBytes = false) {
+export function renderSize(size) {
     // size: number of bytes
-    // includeBytes: whether or not to include 'bytes' string in the return value
     // Return a human-readable string.
     var units = ['', 'k', 'm', 'g', 't'];
     for (var i = 0; i < units.length; i++) {
-        const unit = includeBytes && units[i] === '' ? ' bytes' : units[i];
+        const unit = units[i] === '' ? ' bytes' : units[i];
         if (size < 100 && size !== Math.floor(size)) return Math.round(size * 10) / 10.0 + unit;
         if (size < 1024) return Math.round(size) + unit;
         size /= 1024.0;
     }
 }
 
-export function renderFormat(value, type, includeBytes = false) {
+/**
+ * Converts a list to a single string containing its values.
+ *
+ * @param {array} list
+ * @returns {string} joined list values
+ */
+export function renderList(list) {
+    if (!list?.length || !list[0]) {
```

Comment:
Why `!list[0]`?  In general, I'd like to be a bit more strict about the differences between falsy values...

Reply:
The `tags` field returns `['']` when there aren't any tags. Didn't want to change that in case doing so would break a bunch of stuff, but can change if you think I should.

Reply:
Oh, I see...maybe write that down as a comment - this is something we should fix.  No tags should be `[]`!